vizro-core/src/vizro/models/_page.py

```diff
@@ -162,10 +162,10 @@ def _create_theme_switch():
     def _create_control_panel(controls_content):
         keyline = html.Div(className="keyline")
         control_panel = html.Div(
-            children=[*controls_content, keyline],
+            children=[keyline, *controls_content, keyline],
             className="control_panel",
         )
-        return control_panel if controls_content else None
+        return control_panel if controls_content else keyline
```

Comment:
If we only have one page and no controls there will be random keyline on the left-side no? Did we try to add it at the end of the nav panel container instead of the controls container?

Reply:
Added at the end of the nav_panel. I had to remove nav_panel gap in order for the last accordion-header(title) to look symmetrical to the other accordion headers.